.github/workflows/ci.yml

```diff
@@ -175,6 +178,41 @@ jobs:
       - name: Test
         run: ./ci/scripts/run-english-breakfast-verilator-tests.sh
 
+  # Build CW305 variant of the English Breakfast toplevel design using Vivado
+  chip_englishbreakfast_cw305:
+    name: CW305's Bitstream
+    runs-on: ubuntu-22.04-bitstream
+    needs: quick_lint
+    steps:
+      - uses: actions/checkout@v4
+      - name: Install dependencies
+        uses: ./.github/actions/install-deps
+      - name: Build bitstream
+        run: |
+          . util/build_consts.sh
```

Comment:
This line can be dropped, the script that needs it (`build-bitstream-vivado`) already sources it itself